document/AnnexE-Determinism

```diff
@@ -0,0 +1,52 @@
+# Annex E (informative) Producing deterministic SARIF log files { - #Annex-E }
```

Comment:
you didn't mention ordering in this annex. results or other items which are emitted in a different order may be perceived as different by naive diffing mechanisms (such as source control systems).